Libraries/LibJS/Tests/logical-expressions-basic.js

```diff
@@ -55,7 +54,6 @@ try {
     assert(("foo" || true) === "foo");
     assert((false || "bar") === "bar");
     assert((true || "bar") === true);
-    assert((null || true) === true);
```

Comment:
This should still work, no? Why are we removing it?

Reply:
Same here :)